src/Tizen.Multimedia.Vision/MediaVision/ImageClassificationResult.cs

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Provides the ability to get the result of face detection using <see cref="InferenceModelConfiguration"/> and
+    /// <see cref="ImageClassifier"/>.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ImageClassificationResult
+    {
+        internal ImageClassificationResult(int[] indices, string[] names, float[] confidence, int size)
+        {
+            if (size > 0)
+            {
+                Number = size;
+                Indices = new int[size];
+                Names = new string[size];
+                Confidences = new float[size];
+
+                indices.CopyTo(Indices, 0);
+                names.CopyTo(Names, 0);
+                confidence.CopyTo(Confidences, 0);
+            }
+        }
+
+        /// <summary>
+        /// Gets the number of detected objects.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int Number { get; }
+
+        /// <summary>
+        /// Gets the indices of detected objects.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int[] Indices { get; }
```

Comment:
```suggestion
        public IList<int> Indices { get; }
```

Reply:
I refered https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1819-properties-should-not-return-arrays?view=vs-2017.
I return copied array so I think it's OK.

Is there any other reason you recommend `IList` rather than array itself?

Reply:
✓ DO prefer using collections over arrays in public APIs. The Collections section provides details about how to choose between collections and arrays.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/arrays

Reply:
How about `ReadOnlyCollection<T>`?

✓ DO use ReadOnlyCollection<T>, a subclass of ReadOnlyCollection<T>, or in rare cases IEnumerable<T> for properties or return values representing read-only collections.
In general, prefer ReadOnlyCollection<T>. 
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections\

Reply:
Yes, `ReadOnlyCollection<T>` could be better